services/QuillLMS/app/controllers/teachers/progress_reports/concepts/students_controller.rb

```diff
@@ -9,10 +9,12 @@ def index
   end
 
   private def json_payload
-    {
-      students: students_as_json,
-      classrooms_with_student_ids: current_user.classrooms_i_teach_with_student_ids
-    }
+    current_user.all_classrooms_cache(key: 'teachers.progress_reports.conepts.students') do
```

Comment:
Looks like there is a typo with `conepts`

Reply:
Good catch.